src/components/implementation/no_interface/new_boot/ppos_booter.c

```diff
@@ -80,6 +80,61 @@ boot_find_cobjs(struct cobj_header *h, int n)
 	       hs[n-1]->name, hs[n-1]->id, hs[n-1], cobj_sect_get(hs[n-1], 0)->vaddr);
 }
 
+static int
+boot_comp_map_memory(struct cobj_header *h, spdid_t spdid, pgtblcap_t pt)
+{
+	int i, j;
+	int flag;
+	vaddr_t dest_daddr, prev_map = 0;
+	int tot = 0, n_pte = 1;	
+	struct cobj_sect *sect = cobj_sect_get(h, 0);
+	
+	/* Expand Page table, could do this faster */
+	for (j = 0 ; j < (int)h->nsect ; j++) {
```

Comment:
You have two different counting variables, i and j. You should be able to just use one. You use i below.